StoreHouse/StoreHouse/ClientManagerAddOrderSecondStepView.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Model;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace View
+{
+    public partial class ClientManagerAddOrderSecondStepView : Form, IClientManagerAddOrderSecondStepView
+    {
+        private readonly ApplicationContext _context;
+        public ClientManagerAddOrderSecondStepView(ApplicationContext context)
+        {
+            _context = context;
+            InitializeComponent();
+            this.StartPosition = FormStartPosition.CenterScreen;
+
+        }
+
+        public event Action AddProduct;
+        public event Action Cancel;
+        public event Action Back;
+
+        private void btn_Back_Click(object o, EventArgs e)
+        {
+            Back?.Invoke();
+        }
+
+        private void btn_Cancel_Click(object o, EventArgs e)
+        {
+            Cancel?.Invoke();
+        }
+
+        private void btn_AddProduct_Click(object o, EventArgs e)
+        {
+            AddProduct?.Invoke();
+        }
+
+        public new void Show()
```

Comment:
move this method before `private` methods. And follow next order
![image](https://user-images.githubusercontent.com/36641976/68540516-77580b00-03a4-11ea-9adb-8dea2fa742d5.png)
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods

Reply:
and why `new`?